### ISSUE TITLE
ui-grid end editing without blur

### ISSUE THREAD

**Issue by fgaonet** [angular-ui/ui-grid#5283]
Please make sure each of the following is true before submitting a new issue.  We are a small team with limited time and it is very difficult to manage issues if they aren't reported correctly.

- [ x] This is a bug report, not a question on how to use the grid.  
Use [Stack Overflow](http://stackoverflow.com/questions/tagged/angular-ui-grid) or [Gitter](https://gitter.im/angular-ui/ui-grid) for questions.
- [x ] You have searched the open issues to see if this bug has been filed before.  We don't want duplicate issues.
- [x ] You have reviewed the tutorials and documentation to ensure you are using the grid correctly. [ui-grid.info](http://ui-grid.info/docs/#/tutorial)
- [x ] You are using the latest version of the grid. Older versions are not patched or supported.
- [x ] You have provided steps to recreate your bug.  A plunkr is even better.

**Comment by fgaonet** [comment]
I am using an ui-grid with cell edit enabled. I applied a editableCellTemplate as:
'input type="number" ui-grid-editor ng-model="MODEL_COL_FIELD" /'

I need to raise update on "change" instead of "blur". I tried many way on the template, but still have to blur the input to get update.

Is there any simple way to override the "blur" to "change"? It is default update behavior on Angularjs input, but it seems not working within angular ui-grid.

Thanks,

**Comment by stevenlius** [comment]
Did you try ng-model-options?

**Comment by fgaonet** [comment]
Yes. I tried it, but no help. Actually, the default behavior of ng-input in my template should be "update on change". I used other similar input on my page (out of ui-grid), and all of them "update on change" by default.

I think ui-grid cellEdit fires update on blur by default. I just don't know how to override this default behavior on ui-grid.

Any idea?

Thanks,

**Comment by Novitoll** [comment]
+1

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
